src/js/AgeVerification.tsx

```diff
@@ -0,0 +1,196 @@
+                "https://i.imgur.com/RJ8FNDb.jpg",
+                "https://i.imgur.com/gKhQO5E.jpg",
+                "https://i.imgur.com/Vz7yaEy.jpg",
+                "https://i.imgur.com/1JsqDGw.jpg",
+                "https://i.imgur.com/S29mzqv.jpg",
+            ]
+        }, options);
+
+        if(!options.branding?.name) throw new Error("Company name is required");
+        else if(!options.branding?.logo) throw new Error("Company logo is required");
+        else if(!options.branding?.links?.cookies) throw new Error("Link to company cookie policy is required");
+        else if(!options.branding?.links?.termsOfService) throw new Error("Link to company terms of service is required");
+
+        // return immediately if the user is on cookie or TOS page
+        if((() => {
+            const
+              cookieUrl = new URL((<a href={options.branding.links.cookies}/>).href),
+              tosUrl = new URL((<a href={options.branding.links.termsOfService}/>).href);
+
+            if(cookieUrl.origin === window.location.origin && cookieUrl.pathname === window.location.pathname) return true
+            else if(tosUrl.origin === window.location.origin && tosUrl.pathname === window.location.pathname) return true
+
+            return false;
+        })()) return
+
+        // return immediately if the user has already agreed
+        if(Cookies.get("verifiedAge") === "1" && !DEV_ENV) return
+
+        // first language in array as default
+        let lang = Object.values(langs.languages)[0];
+
+        // use website's primary language
+        if(langs.languages[options.primary_language]) lang = langs.languages[options.primary_language];
+
+        // use computers language if available
+        // @ts-ignore
+        const userLang = ((navigator.language) ? navigator.language : navigator.userLanguage).split("-")[0].toUpperCase();
+        if(langs.languages[userLang]) lang = langs.languages[userLang];
+
+
+        return new Promise((topLevelResolve) => {
+            const container : HTMLDivElement = <div class="av-container"/>;
+            if(options.backgroundBlur) container.classList.add("av-blur");
+            const modal : HTMLDivElement = <div class="av-modal"/>;
+
+            container.append(modal);
+            document.body.append(container);
+
+            if(options.disableScrolling) document.body.classList.add("av-no-scroll");
+
+            container.style.setProperty("--primaryColor", options?.styling?.primaryColor || "red");
+
+            (() : Promise<void> => {
+                return new Promise((resolve) => {
+                    let build = () => {}
+
+                    const langSelector = (() => {
+                        if(Object.keys(langs).length < 2) return "";
+
+                        return <p>
+                            {Object.keys(langs.languages).map((key) => {
+                                return <a href="javascript:void(0)" onClick={() => {
```

Comment:
It's needed for accessibility, If I add an anchor tag it will manipulate the URL which I'm trying to avoid. The only ways to make an anchor tag clickable, is by adding a URL or javascript. This javascript does nothing. Without it, it doesn't render as clickable, and also doesn't allow user to tab to it.